### DISCUSSION TITLE
Optuna dashboard on aws sagemaker notebook instance

### DISCUSSION THREAD

**Discussion by axelbojrup** [optuna/optuna-dashboard#804] [Q&A]
Hi! How can I enable Optuna dashboard on aws sagemaker notebook instance? I have followed the instructions on https://optuna-dashboard.readthedocs.io/en/latest/getting-started.html#command-line-interface but I find no option to launch the dashboard after installation.

**Comment by c-bata** [comment]
Sorry, I've never used AWS SageMaker notebook instances. I'm not sure but probably it does not provide the way to access to the specific port that Optuna Dashboard serves, so you should create a RDS instance and store the optimization history using Optuna's RDBStorage.
